### PR TITLE
Co2 Signal (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 

image

image

image

image

- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.

image

image

image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        "summary": "Get latest by geographic coordinate",
+        "description": "Get latest by geographic coordinate",
+        "operationId": "GetLatestbyLatLon",
+        "parameters": [
+          {
+            "name": "lon",
+            "in": "query",
+            "required": false,
+            "type": "number",
+			"description" : "Longitude",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json:151]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+			"x-ms-summary" : "Latitude"
+          },
+          {
+            "name": "lat",
+            "in": "query",
+            "required": false,
+            "type": "number",
+			"description" : "Latitude",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json:159]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CO2 Signal",
+    "description": "Co2 Signal intent is to allow users to consume electricity when its carbon footprint is lowest. \n\nBy using electricity at the right time, your device favors low-carbon sources of electricity",
+    "version": "1.0",
+	"contact": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json:7]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/CO2 Signal/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiProperties.json:18]: @paulculmsee You have agreed to not have this iconBrandColor for your connector. And there is a default iconBrandcolor for all the independent publisher connectors, i.e. `"#da3b01"`, Please refer the [templates](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json). 

image

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/CO2 Signal/apiProperties.json (hunk 4 above)
Comment (paulculmsee) [reply independent-publisher-connectors/CO2 Signal/apiProperties.json:18]: Sorry misread it and assumed 007ee5 was valid

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json (hunk 3 above)
Comment (paulculmsee) [reply independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json]: Sorted - not sure how I missed this

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**